test/server/http/admin_test.cc

```diff
@@ -56,6 +56,13 @@ using testing::StartsWith;
 namespace Envoy {
 namespace Server {
 
+namespace {
+void expectEqualSerialized(const std::string& str1, const std::string& str2) {
+  EXPECT_TRUE(
+      TestUtility::protoEqual(TestUtility::jsonToStruct(str1), TestUtility::jsonToStruct(str2)));
+}
```

Comment:
nit: Can you move this into TestUtility with a better comment on what it does? This is duplicated in two places so far.